sources/api/routes/users/validate/index.ts

```diff
@@ -0,0 +1,11 @@
+import { Router } from "express";
+import * as AsyncHandler from 'express-async-handler';
+
+import * as UserController from '../../../controllers/UserController';
+
+const router = Router();
+
+router.get('/:uuid', AsyncHandler(UserController.confirmUserEmail));
```

Comment:
`/users/validate` should be the route for the user named "validate". It should not be used for email validation

I suggest another route for this:
- `/auth/validate-email/{uuid}`
- `/validate-email/{uuid}`
- `/validate-user/{uuid}`